.changeset/seven-trains-lie.md

```diff
@@ -0,0 +1,37 @@
+---
+"@biomejs/biome": minor
+---
+
+The `--reporter=summary` has been greatly enhanced. It now shows the list of files that contains violations, the files shown are clickable and can be opened from the editor.
+
+Below an example of the new version:
+
+```
+reporter/parse ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  i The following files have parsing errors.
+
+  - index.css
+
+reporter/format ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  i The following files needs to be formatted.
+
+  - index.css
+  - index.ts
+  - main.ts
```

Comment:
How does this work if you're trying out Biome in a new repository that isn't formatted yet? If there are hundreds (or thousands) of files here, it wouldn't be a summary anymore...